src/App.jsx

```diff
@@ -1,13 +1,16 @@
 import { Routes, Route } from "react-router-dom";
 import LoginScreen from "./pages/SampleLoginScreen";
-
+import Forgotpassword from "./pages/ForgotScreen";
+import ResetPassword from "./pages/ResetPasswordScreen";
 
 //Define your routes for APP here
 
 const App = () => {
   return (
     <Routes>
       <Route path="/" element={<LoginScreen />}></Route>
+      <Route path="/forgot-password" element={<Forgotpassword />}></Route>
```

Comment:
forgot/password
change/password

Reply:
I have changed the router name as you said

forgot/password
change/password